### ISSUE TITLE
Shouldn't be installed as dev dependency?

### ISSUE THREAD

**Issue by Harukisatoh** [goatandsheep/react-native-dotenv#222]
I looked into README, and the command to install `react-native-dotenv` is `npm install react-native-dotenv`, but why is that? Shouldn't be `npm install react-native-dotenv --save-dev`? Or is there a specific need to install it as production dependency?

**Comment by chungweileong94** [comment]
@Harukisatoh It doesn't really matter that much on RN, but it should be installed in "dependencies" instead of "devDependencies", so how I see it is that we do import code from `react-native-dotenv`

**Comment by dwidc** [comment]
@Harukisatoh, @chungweileong94 

I tried installing this react-native-dotenv on devDependencies, and it works just fine.

I use `yarn add -D react-native-dotenv` when install.

Added plugins on `babel.config.js`:
```js
[
  'module:react-native-dotenv',
  {
    moduleName: '@config',
    path: '.config',
  },
],
```

added typescript `config.d.ts`:
```ts
declare module '@config' {
  export const CONFIG: string;
}
```

and import it on my tsx component using
```ts
import { CONFIG } from '@config';
```

everything works. 

So yeah, it should be on dev dependency, no need to include it as runtime lib.
